domains/akshaybhat.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "akshubhat",
+        "email": "akshaybhat2012@gmail.com"
+    },
+    "record": {
+        "CNAME": "akshaybhat.is-a.dev"
```

Comment:
cant use your is-a.dev domain that you want in CNAME

Reply:
@CuteDog5695 What should I use ? Please give random examples

Reply:
anything your website is hosted (if you dont have one you can use Github Pages or Cloudflare Pages to host it)